engines/bastion_katello/app/assets/javascripts/bastion_katello/content-hosts/bulk/views/content-hosts-bulk-system-purpose-modal.html

```diff
@@ -0,0 +1,77 @@
+<div data-extend-template="components/views/bst-modal.html">
+  <h4 data-block="modal-header" translate>Content Host System Purpose</h4>
+
+  <div data-block="modal-body">
+    <h4 translate>Assign System Purpose:</h4>
+
+    <div class="row">
+      <div class="col-sm-12">
+        <div bst-global-notification></div>
+      </div>
+    </div>
+
+    <form name="contentHostContentForm" class="form" ng-hide="content.workingMode">
+      <div>
+        <label translate>Service Level:</label>
+
+        <select type="select"
+                ng-options="item for item in defaultServiceLevels"
+                ng-model="selectedServiceLevels">
+        </select>
+      <br /><br />
+
+        <label translate>Usage Type:</label>
+
+        <select type="select"
+                readonly="denied('edit_hosts', host)"
+                ng-options="item for item in defaultUsages"
+                ng-model="selectedRoles">
+        </select>
+        <br /><br />
+
+        <label translate>Role:</label>
+
+        <select type="select"
+                readonly="denied('edit_hosts', host)"
+                ng-options="item for item in defaultRoles"
+                ng-model="selectedUsages">
+        </select>
+        <br /><br />
+
+        <label ng-hide="purposeAddonsList.length === 0" translate>Add ons:</label>
+        <select multiple ng-multiple="true"
+                ng-hide="purposeAddonsList.length === 0"
+                readonly="denied('edit_hosts', host)"
+                ng-options="item for item in purposeAddonsList"
```

Comment:
```suggestion
                ng-options="item for item in purposeAddonsList()"
```

I _think_ this will work, but angular is doing some magic here..

Reply:
Updated